### PR TITLE
Dotnetcore

### PR DESCRIPTION
**Pull request by erik-inkapool**
Add support for ASP.NET Core RTM.

There's a new sample project demonstrating the differences in how to use the package. The README is currently not updated, would you like me to do that as well?

The diff is pretty rough to read due to my auto formatting options. Sorry about that.

### CODE REVIEW

File: samples/VS2015/SpecFlow 1.9.0/net461 1.0.0/Sample.Website.Tests/project.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+  "dependencies": {
+    "Sample.Website": "1.0.0-*",
+    "SpecFlow": "2.1.0",
+    "xunit": "2.2.0-beta2-build3300",
+    "dotnet-test-xunit": "2.2.0-*",
+  },
+
+  "tools": {
+    "dotnet-specflow": "1.0.0-beta4"
```
Comment (erik-lundgren) [inline samples/VS2015/SpecFlow 1.9.0/net461 1.0.0/Sample.Website.Tests/project.json]: We're now a dotnet tool, so you need to list the dependency in the tool-part of project.json.

File: src/dotnet-specflow/project.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
+﻿{
+  "version": "1.0.0-beta3",
```
Comment (erik-lundgren) [inline src/dotnet-specflow/project.json]: bumped version

File: src/dotnet-specflow/Fixer.cs [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+            });
+            if (dnxUser == null)
+            {
+                throw new Exception("Can't find a user with a .dnx folder in his/her home directory");
+            }
+
+            var SpecFlowPaths = new List<string>
+            {
+                Path.Combine(dnxUser, @".nuget\packages\SpecFlow\2.1.0\tools\specflow.exe"),
+                Path.Combine(dnxUser, @".dnx\packages\SpecFlow\2.1.0\tools\specflow.exe"),
```
Comment (erik-lundgren) [inline src/dotnet-specflow/Fixer.cs]: SpecFlow 2.1.0 now supported.

File: src/dotnet-specflow/Fixer.cs [hunk 4]
```diff
@@ -0,0 +1,269 @@
...
+
+        // TODO: Allow specifying other unit test providers.
+        private const string AppConfigSpecFlowSection = @"	<specFlow>
+		<" + AppConfigSpecFlowSectionElement + @" name=""xUnit"" />
+	</specFlow>";
+
+        public Fixer()
+        {
+            // TODO: Allow for other DNX install locations.
+            var users = Directory.GetDirectories(@"C:\Users");
```
Comment (erik-lundgren) [inline src/dotnet-specflow/Fixer.cs]: There is no longer an `Environment.SpecialFolder.UserProfile` so this is hardcoded now.

File: src/dotnet-specflow/project.json [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+  "description": "Generate tests from SpecFlow feature files inside ASP.NET Core projects (.xproj's).",
+  "authors": [ "stajs" ],
+
+  "packOptions": {
+    "tags": [ "SpecFlow", "ASP.NET Core", "ASP.NET 5", "Visual Studio 2015", "xproj" ],
+    "projectUrl": "https://github.com/stajs/SpecFlow.Dnx",
+    "licenseUrl": "https://github.com/stajs/SpecFlow.Dnx/blob/master/LICENSE"
+  },
+
+  "buildOptions": {
```
Comment (erik-lundgren) [inline src/dotnet-specflow/project.json]: needed since we are a tool.

File: src/SpecFlow.Dnx/Fixer.cs [hunk 6]
```diff
@@ -1,238 +1,269 @@
...
+        public Fixer()
+        {
+            // TODO: Allow for other DNX install locations.
+            var users = Directory.GetDirectories(@"C:\Users");
+
+            var dnxUser = users.SingleOrDefault(u =>
+            {
+                try
+                {
+                    return Directory.GetDirectories(u).Any(d => d.Contains(".dnx"));
```
Comment (stajs) [inline src/SpecFlow.Dnx/Fixer.cs:38]: I doubt we can rely on this folder for people new to .NET Core (i.e. who don't have legacy DNX folders hanging around). Can fix it up later...

**Comment by stajs** [comment]
A big thanks for adding this support! If you can update the README that would be amazeballs :confetti_ball: 

I want to do a rename (#21) before getting up on nuget.org...

**Pull request merged**